projects/ngx-image-gallery/src/lib/ngx-image-gallery.scss

```diff
@@ -5,22 +5,28 @@
   
   .thumbnail {
     width: calc(10% - 10px);
-  }
-  
-  .thumbnail:hover {
-    cursor: pointer;
-  }
-}
 
-.center-content {
-  justify-content: center;
-  align-items: center;
+    &:hover {
+      cursor: pointer;
+    }
+  }
 }
 
-.content-direction-row {
-  flex-direction: row;
-}
+.content {
+  &-center {
+    justify-content: center;
+    align-items: center;
+  }
 
-.content-direction-column {
-  flex-direction: column;
+  &-right {
+    justify-content: end;
```

Comment:
I think you missed `align-items: end;` here. Without it when display images as column and align to right elements still displayed on the left side.